### PR TITLE
006-keywords revisions

### PR DESCRIPTION
**Pull request by amilan17**

### CODE REVIEW

**Comment by amilan17** [comment]
for discussion: 

- [ ] what does this mean: " compliance to WCMP 1.3 defines a set of rules that apply to different
sets of keywords"

**Comment by josusky** [comment]
> In addition to measure well-defined references to dictionaries and keyword types, compliance to WCMP 1.3 defines a set of rules that apply to different sets of keywords that are not included in this measurement.
My long-term memory is poor, but it seems to me that this sentence might have been a reference to KPI-9, where the "scope of distribution" is expressed as a keyword. So here in KPI 6 we only check that it has the correct form (thesaurus etc.) but in KPI-9 we check also the content (e.g. controlled vocabulary).

**Comment by amilan17** [comment]
*    [ ]  need to describe scoring better

File: kpi/006-keywords.adoc [hunk 1]
```diff
@@ -26,31 +22,26 @@ can be user-defined or specified from controlled vocabularies.
 |===
 |Rule |Score
 
-a|`gmd:keywords` is present
+|6.1: There are one to many `keyword` elements present.
```
Comment (tomkralidis) [inline kpi/006-keywords.adoc:25]: `gmd:keyword`

File: kpi/006-keywords.adoc [hunk 2]
```diff
@@ -26,31 +22,26 @@ can be user-defined or specified from controlled vocabularies.
...
 |1
 
-a|`gmd:type` is present
+a|6.2: The `MD_KeywordTypeCodeType` is present.
 
 |1
 
-a|`gmd:thesaurusName` is present
+a|6.3: The `title` element for the thesuarus name is present.
```
Comment (tomkralidis) [inline kpi/006-keywords.adoc:32]: `gmd:title`

File: kpi/006-keywords.adoc [hunk 3]
```diff
@@ -26,31 +22,26 @@ can be user-defined or specified from controlled vocabularies.
...
 |1
 
-a| Keywords terms and thesaurus are present via `gmx:Anchor`
+a|6.4: Keywords and thesaurus names are implemented in the `Anchor` element .
```
Comment (tomkralidis) [inline kpi/006-keywords.adoc:35]: `gmx:Anchor`

File: kpi/006-keywords.adoc [hunk 4]
```diff
@@ -61,29 +52,33 @@ Examples of controlled vocabularies:
...
 
-==== Well defined keywords examples
+==== WMO References
 
-Used keyword references URL of a code registry using an Anchor element.
+* 5.8.1.8 Descriptive keywords
+
+==== XML Examples
+
+The `keyword` value is included in an `Anchor` element with a resolvable HTTP URL.
```
Comment (tomkralidis) [inline kpi/006-keywords.adoc:62]: `gmd:keyword` && `gmx:Anchor`

File: kpi/006-keywords.adoc [hunk 5]
```diff
@@ -61,29 +52,33 @@ Examples of controlled vocabularies:
...
     <gmd:keyword>
-     <gmx:Anchor xlink:href="http://wis.wmo.int/2012/codelists/WMOCodeLists.xml#WMO_CategoryCode_meteorology">meteorology</gmx:Anchor>
+     <gmx:Anchor 
+        xlink:href="http://wis.wmo.int/2012/codelists/WMOCodeLists.xml#WMO_CategoryCode_meteorology">meteorology<gmx:Anchor>
     </gmd:keyword>
   </gmd:MD_Keywords>
 ```
 
-The type of keyword is given in `MD_KeywordTypeCode` element, the "codelist" indicates URL of the code list, e.g. https://standards.iso.org/iso/19139/resources/gmxCodelists.xml#MD_KeywordTypeCode_theme.
+The `type` of keyword is given in `MD_KeywordTypeCode` element, the "codelist" indicates URL of the code list.
```
Comment (tomkralidis) [inline kpi/006-keywords.adoc:73]: `gmd:type`

**Review by tomkralidis — CHANGES_REQUESTED** [review]

**Review by tomkralidis — APPROVED** [review]

**Pull request merged**